From df66d760a2fd229d1989ed9343693ba2ad9a8964 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 20 Jun 2022 09:57:44 +0200
Subject: [PATCH 220/529] dt-bindings: spi: stm32: add address-cells and
 size-cells into yaml

Theses properties need to be described to satisfy dtbs_check.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I05058484ccf1c30e842d6bd840b47b3efbb1a36b
---
 Documentation/devicetree/bindings/spi/st,stm32-spi.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
+++ b/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
@@ -29,6 +29,9 @@ allOf:
         st,spi-midi-ns: false
 
 properties:
+  "#address-cells": true
+  "#size-cells": true
+
   compatible:
     enum:
       - st,stm32f4-spi
